docs/content/concepts/partitions-schedules-sensors/partitioning-assets.mdx

```diff
@@ -216,6 +216,12 @@ If using the default I/O manager, materializing partition `2022-07-23` of this a
 
 ---
 
+## Recommended partition limits
```

Comment:
Thoughts on putting this in the **Defining partitioned assets** section? It feels a bit oddly placed here, and I'm concerned it might get lost.

Spitballing some suggestions:

- Add the info to the intro section of **Defining partitioned assets** as a callout
- Add the info to the intro section of **Defining partitioned assets**, between the first paragraph (explanation) and the second (introduces the example)
- Make the info the last subsection (H3) in **Defining partitioned assets**, so the structure would look like this:

```
- Defining partitioned assets
  - Multi-dimensionally partitioned assets
  - Dynamically partitioned assets
  - Partitioned assets with partitioned I/O managers
  - Recommended partition limits
```

Reply:
Good call on putting this within the defining partitioned assets section.

Adding the info as the last subsection sounds good. I think being able to reference this content via the section in the sidebar is helpful for searching.